docs/contributing.md

```diff
@@ -101,5 +101,6 @@ git commit
 
 ### Project development go tools
 
-The project uses [Bingo](.bingo/README.md) for adding any go tools required by the project.
+The project uses [Bingo](https://github.com/tellor-io/telliot/blob/master/.bingo/README.md) for adding any go tools required by the project.
```

Comment:
can we keep it relative here and just replace any relative links with absolute paths in the sync with the main docs?

Reply:
There are page links that need to stay relative. this could result in broken links in the TellorDocs repo!

Reply:
all right I see what you mean so yeah lets leave it like this then.